examples/sighup-example.rs

```diff
@@ -0,0 +1,33 @@
+extern crate futures;
+extern crate tokio_core;
+extern crate tokio_signal;
+
+use futures::stream::Stream;
+use tokio_core::reactor::Core;
+use tokio_signal::unix::{Signal,SIGHUP};
+
+fn main() {
+    // set up a Tokio event loop
+    let mut core = Core::new().unwrap();
+
+    // on Unix, we can listen to whatever signal we want, in this case: SIGHUP
+    let sighup = Signal::new(SIGHUP, &core.handle());
+    let stream = core.run(sighup).unwrap();
```

Comment:
No worries! I think you just need to import `Future` above to get access to the `flatten_stream()` method. I'll merge and clean that up locally.